pages/guide/01-quickstart.md

```diff
@@ -258,8 +258,6 @@ effects, but for now we'll use a community package called
 every time the counter is incremented.
 
 Because this is a separate package, make sure to add it to your project first.
-While we're here, we'll also add `gleam_json` so we can decode the response from
```

Comment:
`gleam_json` is never added more directly used in the tutorial. It looks like `lustre_http` depends on `gleam_json` so I've removed this.